charts/opensearch-cluster/README.md

```diff
@@ -0,0 +1,24 @@
+# OpenSearch-k8s-operator
```

Comment:
There needs to be some documentation about this chart in the main userguide.
And I think this file should be moved to the `docs/userguide` folder.

Reply:
Please also extend this to give more details on how the chart can be used (what is there now are generic helm usage basics but nearly nothing specific of the chart and what you can configure).